plugins/tech-radar/src/utils/components.tsx

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import React from 'react';
+
+type WithLinkProps = {
+  url?: string;
+  className: string;
+  children: React.ReactNode;
+};
+
+export const WithLink = ({
```

Comment:
Couldn't this use `React.FC` instead?

For example

```tsx
type Props = {
  url?: string;
  className: string;
};

export const WithLink: React.FC<Props> = ({ url, className, children }) => {
  ...
};
```

Reply:
#1376 x)